css/localgov-alert-banner.css

```diff
@@ -4,52 +4,74 @@
  */
 
 /* Default */
-.localgov_alert_banner {
+.localgov-alert-banner {
+  padding-top: 1rem;
+  padding-bottom: 1rem;
```

Comment:
Not sure about the extra padding. Given alert banners tend to be compacted when viewed (see examples on Croydon.gov.uk etc...) then this would be a style that almost every council has to override. May be better to remove it as default.

Reply:
That padding is removed now.